conda/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+    name: odlcuda
+    version: "0.3.0"
+
+source:
+    git_url: https://github.com/odlgroup/odlcuda.git
+    git_rev: conda-build
+
+build:
+    script_env:
+        - CUDA_COMPUTE
+
+requirements:
+    build:
+        - python 2.7|>=3.4
+        - nomkl
+        - setuptools
+        - numpy >=1.9
```

Comment:
It can probably be removed, but since it's a direct dependency I'm reluctant to remove it.